plugins/scaffolder-common/src/permissions.ts

```diff
@@ -78,12 +91,46 @@ export const templateStepReadPermission = createPermission({
   resourceType: RESOURCE_TYPE_SCAFFOLDER_TEMPLATE,
 });
 
+/**
+ * This permission is used to authorize actions that involve reading one or more tasks in the scaffolder,
+ * and reading logs of tasks
+ *
+ * @alpha
+ */
+export const taskReadPermission = createPermission({
+  name: 'scaffolder.task.read',
+  attributes: {
+    action: 'read',
+  },
+});
+
+/**
+ * This permission is used to authorize actions that involve the creation of tasks in the scaffolder.
+ *
+ * @alpha
+ */
+export const taskCreatePermission = createPermission({
+  name: 'scaffolder.task.create',
+  attributes: {
+    action: 'create',
+  },
+});
+
+/**
+ * This permission is used to authorize actions that involve the cancellation of tasks in the scaffolder.
+ *
+ * @alpha
+ */
+export const taskCancelPermission = createPermission({
+  name: 'scaffolder.task.cancel',
+  attributes: {},
+});
+
 /**
  * List of all the scaffolder permissions
  * @alpha
  */
 export const scaffolderPermissions = [
-  actionExecutePermission,
   templateParameterReadPermission,
   templateStepReadPermission,
```

Comment:
let's list all permissions supported by scaffolder here.

```suggestion
  templateStepReadPermission,
  ...scaffolderActionPermissions,
  ...scaffolderTaskPermissions
```